src/Service.Tests/GraphQLBuilder/MutationBuilderTests.cs

```diff
@@ -94,7 +94,7 @@ private static Entity GenerateEmptyEntity()
             FieldDefinitionNode createField =
                 query.Fields.Where(f => f.Name.Value == $"createFoo").First();
             Assert.AreEqual(expected: isAuthorizeDirectiveExpected ? 1 : 0,
-                actual: createField.Directives.Count);
+                actual: createField.Directives.Where(x => x.Name.Value is "authorize").Count());
```

Comment:
Could have used the constant: `GraphQLUtils.AUTHORIZE_DIRECTIVE`

Reply:
fixed.